### DISCUSSION TITLE
打包显示成功，但是为什么还是这个报错呢？

### DISCUSSION THREAD

**Discussion by 2644635282** [tw93/Pake#624] [Q&A]
[windows-latest](https://github.com/2644635282/Pake_Web/actions/runs/7238404691/job/19719178136#step:12:11)
No files were found with the provided path: node_modules/pake-cli/output/*. No artifacts will be uploaded.

![微信截图_20231217202756](https://github.com/tw93/Pake/assets/62086344/f8827f1c-1e5a-4c10-90c2-c7fbd27cc97d)

**Comment by 2644635282** [comment]
希望大佬帮我看看呢

**Reply by tw93** [reply]
谢谢反馈 看起来是之前有一个特性没有处理到位,错误在这里https://github.com/2644635282/Pake_Web/actions/runs/7238404691/job/19719178136#step:11:582

**Reply by 2644635282** [reply]
收到，请问这个问题我应该如何解决？或者是否有交流群之类的大家学习交流的地方呢

**Reply by 2644635282** [reply]
貌似改了仓库命令就会出问题，然后如果打包的名称中有空格，例如“Friday ai”中间有个空格，也会打包失败，不加空格就成功了~

**Comment by 17580** [comment]
同样的问题，显示成功，依然报错～

**Reply by 17580** [reply]
找到问题了，不能使用中文名称。只能英文和数字。参考：https://github.com/tw93/Pake/discussions/626

**Comment by XinMeiHT** [comment]
啥玩意，问题一堆，不好用！
